webrtc.html

```diff
@@ -6089,9 +6089,9 @@ <h2>Attributes</h2>
               associated with this <code><a>RTCRtpSender</a></code> object. If
               <code>track</code> is ended, or if the track's output is disabled,
               i.e. the track is disabled and/or muted, the <code>
-              <a>RTCRtpSender</a></code> MUST send silence (audio), black
-              frames (video) or a zero-information-content equivalent. In the
-              case of video, the <code><a>RTCRtpSender</a></code> SHOULD send one
+              <a>RTCRtpSender</a></code> MUST send black frames (video) or
+              MUST NOT send (audio).  In the case of video,
```

Comment:
Better language might be "In the case of audio, the recommended way to send silence is to send no packets."